manuscript/main.tex

```diff
@@ -624,8 +638,8 @@ \subsection*{Gut microbial profiles in the first year of life predictict future
 could predict \textit{future} cognitive performance (Figure 3B, D).
 Models were consistently able to learn microbial species that are associated
 with cognitive performance at a future visit
-(mean time between visits: XX, %TODO
-mean test set correlation: YY).
+(mean time between visits: 21.5 months [sd = 9.9 months] (Figure 3C),
```

Comment:
Is the time between visits normally distributed? Median time / 5%/95% percentiles might make more sense to report?

Reply:
Will change. Just for reference, the percentiles: 5th = 5.6; 50th = 21.9; 95th = 37.7